x-pack/plugins/ml/public/application/components/job_spaces_repair/repair_results.tsx

```diff
@@ -0,0 +1,178 @@
+          <EuiTextColor color="subdued">
+            <FormattedMessage
+              id="xpack.ml.management.repairSavedObjectsFlyout.savedObjectsDeleted.description"
+              defaultMessage="Saved objects which do not have an accompanying job. These will be left over when deleting anomaly detection jobs."
+            />
+          </EuiTextColor>
+        </p>
+      </EuiText>
+    </>
+  );
+  return <RepairItem id="savedObjectsDeleted" title={title} items={items} />;
+};
+
+const DatafeedsAdded: FC<{ repairItems: RepairSavedObjectResponse }> = ({ repairItems }) => {
+  const items = Object.keys(repairItems.datafeedsAdded);
+
+  const title = (
+    <>
+      <EuiTitle size="xs">
+        <h3>
+          <EuiTextColor color={items.length ? 'default' : 'subdued'}>
+            <FormattedMessage
+              id="xpack.ml.management.repairSavedObjectsFlyout.datafeedsAdded.title"
+              defaultMessage="Job saved objects with missing datafeeds ({count})"
+              values={{ count: items.length }}
+            />
+          </EuiTextColor>
+        </h3>
+      </EuiTitle>
+      <EuiText size="s">
+        <p>
+          <EuiTextColor color="subdued">
+            <FormattedMessage
+              id="xpack.ml.management.repairSavedObjectsFlyout.datafeedsAdded.description"
+              defaultMessage="Anomaly detection job saved objects which are missing their datafeed id."
+            />
+          </EuiTextColor>
+        </p>
+      </EuiText>
+    </>
+  );
+  return <RepairItem id="datafeedsAdded" title={title} items={items} />;
+};
+
+const DatafeedsRemoved: FC<{ repairItems: RepairSavedObjectResponse }> = ({ repairItems }) => {
+  const items = Object.keys(repairItems.datafeedsRemoved);
+
+  const title = (
+    <>
+      <EuiTitle size="xs">
+        <h3>
+          <EuiTextColor color={items.length ? 'default' : 'subdued'}>
+            <FormattedMessage
+              id="xpack.ml.management.repairSavedObjectsFlyout.datafeedsRemoved.title"
+              defaultMessage="Job saved objects with unmatched datafeed ids ({count})"
+              values={{ count: items.length }}
+            />
+          </EuiTextColor>
+        </h3>
+      </EuiTitle>
+      <EuiText size="s">
+        <p>
+          <EuiTextColor color="subdued">
+            <FormattedMessage
+              id="xpack.ml.management.repairSavedObjectsFlyout.datafeedsRemoved.description"
+              defaultMessage="Anomaly detection jobs which contain a datafeed ID, but the datafeed does not exist."
+            />
+          </EuiTextColor>
+        </p>
+      </EuiText>
+    </>
+  );
+  return <RepairItem id="datafeedsRemoved" title={title} items={items} />;
+};
+
+const RepairItem: FC<{ id: string; title: JSX.Element; items: string[] }> = ({
+  id,
+  title,
+  items,
+}) => (
+  <EuiAccordion id={id} buttonContent={title} paddingSize="l">
+    <EuiText size="s">
+      {items.map((item) => (
+        <div key={item}>{item}</div>
+      ))}
```

Comment:
Looks like you're creating a list here! Instead of a series of `div` tag, can we create an HTML list? Something like:

```
{items.length &&
  <ul>
    {items.map((item) => (
        <li key={item}>{item}</li>
    ))}
  </ul>
}
```

Reply:
updated in https://github.com/elastic/kibana/pull/83320/commits/37f041d387e5bc3f7b61e528aca9993aeefd94cb